servicebroker/lib/routes/binding.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+
+module.exports = function(app) {
+  var path = require('path');
+  var fs = require('fs');
+  var request = require('request');
+  var logger = require(path.join(__dirname, '../logger/logger.js'));
+  var models = require(path.join(__dirname, '../models'))();
+
+  var settings = JSON.parse(
+    fs.readFileSync(path.join(__dirname, '../../config/settings.json'), 'utf8'));
+  var apiServerUtil = require(path.join(__dirname, '../util/apiServerUtil.js'))(settings.apiServerUri);
```

Comment:
Couple of comments.
1. This is the second time we have read the settings and we don't do validation anywhere
2. More importantly, we should just inject the apiServerUri to this module. This allows for easier testing.